src/ninshiki_cpp_detector.cpp

```diff
@@ -55,6 +54,9 @@ int main(int argc, char ** argv)
 
   // Handle arguments
   try {
+    if (argc < 2) {
+      throw std::runtime_error("");
```

Comment:
1. Add the error description.
2. I suggests to log to the `std::cerr` and return non-zero instead of throwing an exception.